### PR TITLE
List all cheat sheets in this repository

### LINKED ISSUE #7: List all sheets from latex4ei.de in the README
Motivated by https://github.com/latex4ei/Computertechnik/issues/1 and the fact that not everyone can access the website, I thought about managing the website content on GitHub as well.

I created a first draft in https://github.com/latex4ei/Allgemein/blob/master/README.md#formelsammungen, but we would need to add the rest as well. Once this is done, we could update latex4ei.de to point to this repo. Additionally, we could also setup a static page generator, but looking at the many "awesome ..." repos, having a simple markdown could do the job.

@dariusptrs What do you think?

### PR DESCRIPTION
**Pull request by dariusptrs**
closes #7

### CODE REVIEW

**Comment by dariusptrs** [comment]
I had to remove `mdformat-toc` because it was always removing the emojis in the TOC.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -25,4 +27,3 @@ repos:
           - mdformat-black
           - mdformat-shfmt
           - mdformat-config
-          - mdformat-toc
```
Comment (hofbi) [inline .pre-commit-config.yaml]: Keeping this is essential, else we would not autogenerate the TOC in the README

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -5,6 +5,7 @@ repos:
     hooks:
       - id: check-hooks-apply
       - id: check-useless-excludes
+
```
Comment (hofbi) [inline .pre-commit-config.yaml]: Why do we need the extra lines here?

File: README.md [hunk 3]
```diff
@@ -4,61 +4,180 @@ Mitschriften und Formelsammlungen in LaTeX für Elektro- und Informationstechnik
...
-  - [Allgemein](#allgemein)
-  - [1. Semester](#1-semester)
-  - [2. Semester](#2-semester)
-  - [3. Semester](#3-semester)
-  - [4. Semester](#4-semester)
-  - [Externe Formelsammlungen](#externe-formelsammlungen)
-- [Formelsammlungen für Latex4ei](#formelsammlungen-f%C3%BCr-latex4ei)
-- [Optional: Build with CMake](#optional-build-with-cmake)
-  - [Steps to build](#steps-to-build)
+## 📚 Table of Contents
```
Comment (hofbi) [inline README.md]: Move this line above of line 5, then we can use the pre-commit hook to keep the toc down here automatically up to date

File: README.md [hunk 4]
```diff
@@ -4,61 +4,180 @@ Mitschriften und Formelsammlungen in LaTeX für Elektro- und Informationstechnik
...
+| CI Status                                                                              | Formelsammlung                                                                                                 | Repository                                                  |
+| -------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------- |
+| ![CI Status](https://github.com/latex4ei/Analysis-1/workflows/CI/badge.svg)            | [Analysis 1 Formelsammlung](https://github.com/latex4ei/Analysis-1/raw/gh-pages/Analysis-1.pdf)                | [GitHub](https://github.com/latex4ei/Analysis-1)            |
+| ![CI Status](https://img.shields.io/badge/status-not%20available-lightgrey)            | [Analysis 1 für Mathematiker](https://latex4ei.de/downloads/Analysis_1_ma.pdf)                                 | N/A                                                         |
```
Comment (hofbi) [inline README.md]:
```suggestion
| ![CI Status](https://img.shields.io/badge/status-not%20available-lightgrey)            | [Analysis 1 für Mathematiker](https://latex4ei.de/downloads/Analysis_1_ma.pdf)                                 | [GitHub](https://github.com/latex4ei/Analysis-1)                                                          |
```

Both math sheets share the same repo

File: .pre-commit-config.yaml (hunk 1 above)
Comment (dariusptrs) [reply .pre-commit-config.yaml]: okay, what do we have to change so that the emojis do not get removed from the TOC? Add them into the headings?

**Review by hofbi — APPROVED** [review]

**Pull request merged**